barman/infofile.py

```diff
@@ -1020,7 +1021,7 @@ def is_checksum_consistent(self):
 
     def is_full_and_eligible_for_incremental(self):
         """
-        Used to filter out backups that have a parent backup id and are not
+        Used to filter out backups that are incremental and are not
```

Comment:
```suggestion
        Used to filter out backups that incremental backups that are not
```

Reply:
@andremagui i think makes more sense

Reply:
I applied the suggestion made by @barthisrael on the same part of the docstring that sounded clearer to me! I think the **must not be part** is kinda confusing in here.